chromium/rules.js

```diff
@@ -250,20 +250,20 @@ RuleSets.prototype = {
 
     // Read user prefs
     if (rule_set.name in this.ruleActiveStates) {
-      rule_set.active = (this.ruleActiveStates[rule_set.name] == "true");
+      rule_set.active = this.ruleActiveStates[rule_set.name];
     }
 
     var rules = ruletag["rule"];
     for (let rule of rules) {
-      if (rule["from"] != null && rule["to"] != null) {
+      if (rule["from"] && rule["to"]) {
```

Comment:
This code block will not execute in cases when it did before in the case where `rule["from"]` and `rule["to"]` are falsey, but not undefined. e.g:
```javascript
0 != null && false != null  // true
0 && false  // false
```

Reply:
Thanks!

Reply:
Why would `from` and `to` be falsey? They should not be empty.